### PR TITLE
Display real trade data + fix for sell with ztg

### LINKED ISSUE #842: Notification after successful trade should display real values, rather than pre-calculated
Parse `SwapExactAmount...` extrinsics events and display real results of a swap instead of values returned by the functions in `lib/math`

### PR DESCRIPTION
**Pull request by TvrtkoM**
close #842

### CODE REVIEW

File: components/trade-form/index.tsx [hunk 1]
```diff
@@ -27,6 +28,36 @@ import { useExtrinsic } from "lib/hooks/useExtrinsic";
...
+        baseAsset = baseAsset.add(data[0]["assetAmountIn"].toPrimitive());
+        outcome = outcome.add(data[0]["assetAmountOut"].toPrimitive());
+      } else {
+        baseAsset = baseAsset.add(data[0]["assetAmountOut"].toPrimitive());
+        outcome = outcome.add(data[0]["assetAmountIn"].toPrimitive());
+      }
+    }
+  }
+  return {
+    baseAmount: baseAsset.div(ZTG).toFixed(4),
```
Comment (Robiquet) [inline components/trade-form/index.tsx:57]: ZTG is only worth $0.1 so 4 decimal places of precision is a bit excessive, probably 1 is plenty.

**Review by Robiquet — COMMENTED** [review]
Looks good, is it possible to get the same values into `TradeResult`?

File: components/trade-form/index.tsx (hunk 1 above)
Comment (TvrtkoM) [reply components/trade-form/index.tsx:57]: they're in TradeResult - https://github.com/zeitgeistpm/ui/blob/b0352999a15222678e35232e78e754500e0ae688/components/trade-form/index.tsx#L183

**Comment by TvrtkoM** [comment]
> Looks good, is it possible to get the same values into `TradeResult`?

They're in TradeResult

https://github.com/zeitgeistpm/ui/blob/b0352999a15222678e35232e78e754500e0ae688/components/trade-form/index.tsx#L183

https://github.com/zeitgeistpm/ui/blob/b0352999a15222678e35232e78e754500e0ae688/components/trade-form/index.tsx#L434

**Review by Robiquet — APPROVED** [review]
Oh nice, I didn't notice

**Pull request merged**